AutomatedTesting/CMakePresets.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
How does this affect projects made with the engine before the changes in this PR?

Will compilation break for projects that are missing a CMakePresets.json file?

Reply:
This information is in the __How was this PR Tested?__ section in the description.